src/components/__snapshots__/Calculator.test.jsx.snap

```diff
@@ -0,0 +1,204 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Calculator component Matches DOM Snapshot 1`] = `
```

Comment:
- [OPTIONAL] According to the project requirements, There should be the usage of Jest snapshots for testing all the components. however, I only see the Calculator and Route test snapshots 🤔 Please make sure to add Navbar.jsx Jest snapshots test ✍🏻